go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/lightsparkdev/go-sdk
 
-go 1.21
+go 1.22.0
 
-toolchain go1.21.0
+toolchain go1.23.2
```

Comment:
Did this change automatically? I'm not totally sure we want to change these versions.

Reply:
Yeah this changed automatically. It seems this is also causing the LNURL Server action step to fail. Let me revert this!

Reply:
Hmm, I ran `go mod tidy` and it seems to auto update the go version. Could I update go.mod for the LNURL & UMA server instead? Sorry for all the commits!

Reply:
All good! Yeah it should be ok to run `go mod tidy` in those too